expr/function/function.go

```diff
@@ -20,7 +20,50 @@ type Interface interface {
 	Call([]zng.Value) (zng.Value, error)
 }
 
+var deprecated = map[string]string{
+	"Math.abs":              "abs",
+	"Math.ceil":             "ceil",
+	"Math.floor":            "floor",
+	"Math.log":              "log",
+	"Math.max":              "max",
+	"Math.min":              "min",
+	"Math.mod":              "mod",
+	"Math.round":            "round",
+	"Math.pow":              "pow",
+	"Math.sqrt":             "sqrt",
+	"String.byteLen":        "len",
+	"String.formatFloat":    "type cast, e.g., <float-value>:string",
+	"String.formatInt":      "type cast, e.g., <int-value>:string",
+	"String.formatIp":       "type cast, e.g., <ip-value>:string",
+	"String.parseFloat":     "type cast, e.g., <string-value>:float64",
+	"String.parseInt":       "type cast, e.g., <string-value>:int64",
+	"String.parseIp":        "type cast, e.g., <string-value>:ip",
+	"String.replace":        "replace",
+	"String.runeLen":        "rune_len",
+	"String.toLower":        "to_lower",
+	"String.toUpper":        "to_upper",
+	"String.trim":           "trim",
+	"Time.fromISO":          "iso",
+	"Time.fromMilliseconds": "msec and ype cast, e.g., msec(<msec-value>):time",
+	"Time.fromMicroseconds": "usec and type cast, e.g., usec(<usec-value>):time",
+	"Time.fromNanoseconds":  "type cast, e.g., <nsec-value>:time",
+	"Time.trunc":            "trunc",
+	"toBase64":              "to_base64",
+	"fromBase64":            "from_base64",
+}
+
+func isDeprecated(name string) error {
+	msg, ok := deprecated[name]
+	if ok {
+		return fmt.Errorf("function is deprecated: use %s", msg)
```

Comment:
```suggestion
		return fmt.Errorf("function %s is deprecated: use %s", name, msg)
```

Reply:
I had it this way to begin with but the caller prints the function name as part of its error message so the name appeared twice.  I suppose I could change the caller and the other error condition here.  Maybe I'll leave it for this merge and we can change it later if warranted.